projects/1.20/assets/dungeons-delight/dungeons_delight_gourmet_battles/lang/zh_cn.json

```
@@ -0,0 +1,91 @@
+{
+  "advancements.crispyandsweet.descr": "这些腿尝起来怎么样？",
+  "advancements.crispyandsweet.title": "香脆甜美",
+  "advancements.dragon_delight.descr": "你烹饪了末地的守护者",
+  "advancements.dragon_delight.title": "龙之乐事",
+  "advancements.dungeon_apprentice.descr": "准备好开始地下城烹饪之旅了吗？",
+  "advancements.dungeon_apprentice.title": "地下城新手",
+  "advancements.flamingpudding.descr": "甜蜜的痛苦？火热的甜蜜？",
+  "advancements.flamingpudding.title": "火焰布丁",
+  "advancements.ghostlyhanbuger.descr": "快速准备好的恶魂肉！",
+  "advancements.ghostlyhanbuger.title": "恶魂汉堡",
+  "advancements.greenpizzeria.descr": "你做出了意大利奇迹！",
+  "advancements.greenpizzeria.title": "绿色披萨店",
+  "advancements.greenroll.descr": "你做出了地下城奶酪！",
+  "advancements.greenroll.title": "绿色卷",
+  "advancements.gunpowdermeat.descr": "相信不会爆炸！",
+  "advancements.gunpowdermeat.title": "火药肉",
+  "advancements.impracticalpie.descr": "火热且甜蜜的工作",
+  "advancements.impracticalpie.title": "不切实际的馅饼",
+  "advancements.littleflyingsnacks.descr": "好好浸入史莱姆中保存它！",
+  "advancements.littleflyingsnacks.title": "小飞天零食",
+  "advancements.littlepromisingbacons.descr": "紫色培根加鸡蛋的味道尝起来怎么样？",
+  "advancements.littlepromisingbacons.title": "有前途的小培根",
+  "advancements.m_cghost.descr": "美味的白色汉堡",
```

原文`The good old white hamburger`

```suggestion
  "advancements.m_cghost.descr": "经典白色汉堡",
```